app/graphql/resolvers/attachment_metadata_resolver.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module Resolvers
+  # Attachment Metadata Resolver
+  class AttachmentMetadataResolver < BaseResolver
+    type GraphQL::Types::JSON, null: false
+
+    argument :keys, [GraphQL::Types::String],
+             required: false,
+             description: 'Optional array of keys to limit metadata result to.',
+             default_value: nil
+
+    alias attachment object
+
+    def resolve(args)
+      scope = attachment
```

Comment:
You don't need to set the scope to attachment here. This can be removed and the below lines can just be set to use `attachment.metadata`. The reason you see us use `scope` in other resolvers is because we are getting the authorized scope for the model (only the records the user has permissions on)

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/408/commits/9b6cba135b8c37588c4339188902cd42339a59b2